src/ui/server/controllers/insurance/your-buyer/your-buyer-details/index.ts

```diff
@@ -0,0 +1,52 @@
+import { PAGES } from '../../../../content-strings';
+import { Request, Response } from '../../../../../types';
+import { mapCountries } from '../../../../helpers/mappings/map-countries';
+import api from '../../../../api';
+import { FIELD_IDS, ROUTES, TEMPLATES } from '../../../../constants';
+import singleInputPageVariables from '../../../../helpers/page-variables/single-input/insurance';
+
+const FIELD_ID = FIELD_IDS.BUYER_COUNTRY;
+
+export const PAGE_VARIABLES = {
+  FIELD_ID,
+  PAGE_CONTENT_STRINGS: PAGES.YOUR_BUYER,
+};
+
+export const get = async (req: Request, res: Response) => {
+  if (!req.session.submittedData || !req.session.submittedData.insuranceEligibility) {
+    req.session.submittedData = {
+      ...req.session.submittedData,
+      insuranceEligibility: {},
+    };
+  }
+
+  const countries = await api.external.getCountries();
+
+  if (!countries || !countries.length) {
+    return res.redirect(ROUTES.PROBLEM_WITH_SERVICE);
+  }
+
+  let countryValue;
+
+  if (req.session.submittedData && req.session.submittedData.insuranceEligibility[FIELD_IDS.BUYER_COUNTRY]) {
+    countryValue = req.session.submittedData.insuranceEligibility[FIELD_IDS.BUYER_COUNTRY];
+  }
```

Comment:
This won't work for the full application, need to get the country value from `res.locals.application` instead.

This will be empty until we are doing data saving/sending to API in the POST